pkg/gui/containers_panel.go

```diff
@@ -90,6 +94,37 @@ func (gui *Gui) handleContainerSelect(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) renderContainerEnv(container *commands.Container) error {
+	mainView := gui.getMainView()
+	mainView.Autoscroll = false
+	mainView.Wrap = gui.Config.UserConfig.Gui.WrapMainPanel
+	envVariablesList := [][]string{}
+	if len(container.Details.Config.Env) > 0 {
+		for _, env := range container.Details.Config.Env {
+			splitEnv := strings.Split(env, "=")
+			// if the value has = in it, lets say export "test=foo=bar" then split will result in the following
+			// {"test", "foo","bar"} hence join all the elements in the slice except the first one to get value
+			envVariablesList = append(envVariablesList,
+				[]string{
+					utils.ColoredString(splitEnv[0]+":", color.FgBlue),
+					utils.ColoredString(strings.Join(splitEnv[1:], "="), color.FgYellow),
+				})
+		}
+	} else {
+		envVariablesList = append(envVariablesList, []string{"Nothing to display"})
+	}
+	renderedTable, err := utils.RenderTable(envVariablesList)
+	// in case of some error
+	if err != nil {
+		// log the error
```

Comment:
likewise here. I only add comments where I need to explain the something that isn't well communicated by the code itself

Reply:
Removed any unnecessary comments